ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/NotifierSelection.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useEffect, ReactElement } from 'react';
+
+import {
+    Button,
+    ButtonVariant,
+    Flex,
+    FlexItem,
+    FormGroup,
+    SelectOption,
+    TextInput,
+} from '@patternfly/react-core';
+
+import SelectSingle from 'Components/SelectSingle';
+import FormLabelGroup from 'Components/PatternFly/FormLabelGroup';
+import { fetchIntegration } from 'services/IntegrationsService';
+import { NotifierIntegration } from 'types/notifier.proto';
+// eslint-disable-next-line import/no-named-as-default
+import EmailNotifierFormModal from './EmailNotifierFormModal';
+
+type NotifierSelectionProps = {
+    selectedNotifier: NotifierIntegration | null;
```

Comment:
I changed this from just the notifier id to the whole object. I did this because when we go to the `Review and create` section, we want to be able to display the name of the notifier. If we just stored the id then we couldn't do that

Reply:
Also, the formik stuff isn't set up yet, but once I do that, I'll use that properly for this component and all the other form components